delta-api/src/main/java/io/cdap/delta/api/DeltaTargetContext.java

```diff
@@ -47,7 +47,8 @@
    * always have the same corresponding sequence number.
    *
    * Also writes out metrics on the number of DML and DDL operations that were applied since the last successful
-   * commit, as recorded by calls to {@link #incrementCount(DDLOperation)} and {@link #incrementCount(DMLOperation)}.
+   * commit, as recorded by calls to {@link #incrementCount(String, DDLOperation.Type)} and
```

Comment:
There is no "incrementCount(String, DDLOperation.Type)" , should it be incrementCount(DDLOperation.Type) ? a typo?

Reply:
ah good catch! forgot to modify this comment after i modified the method signature. thanks!